### PR TITLE
Pavel/collapsible

### PR DESCRIPTION
**Pull request by galloween**
First draft of Collapsible panel component (no responsive or mobile views)

### CODE REVIEW

File: projects/ui-framework/src/lib/style/style.scss [hunk 1]
```diff
@@ -1,8 +1,5 @@
 @import './variables.scss';
 @import './typography.scss';
-@import './colors.scss';
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/style/style.scss]: why are these removed?

Comment (omrihecht) [reply projects/ui-framework/src/lib/style/style.scss]: the mixins in these files are exposed through this import to other projects, is it somewhere else now?

Comment (galloween) [reply projects/ui-framework/src/lib/style/style.scss]: only colors is removed - its being imported by variables now, because I put the css vars colors to scss variables

Comment (galloween) [reply projects/ui-framework/src/lib/style/style.scss]: oh i did not know the styles file was imported by other projects. i will put back the shapes then, along with the mixins and functions files I created, I guess
